scripts/verify-user-specific-list.sh

```
@@ -0,0 +1,180 @@
+
+# second tools/list — should reuse cached session
+TOOLS_SECOND=$(list_tools "$SESSION_REUSE" "Authorization: Bearer user-a-token")
+SECOND_NAMES=$(echo "$TOOLS_SECOND" | jq -r '[.result.tools[]?.name] | sort | join(",")' 2>/dev/null)
+
+check '[ "$FIRST_NAMES" = "$SECOND_NAMES" ]'  "second tools/list returns identical tool set"
+check 'has_tool "$TOOLS_SECOND" "uspec_list_repos"'  "user-specific tools still present on reuse"
+
+# ── Step 8: MCPServerRegistration status ──────────────────────────────
+info "Step 8: MCPServerRegistration shows ready status"
+
+STATUS_TYPE=$(kubectl get mcpsr -n "$NAMESPACE" test-user-specific-server \
```

Shellcheck (0.11.0)

[warning] STATUS_TYPE appears unused. Verify use (or export if used externally).

(SC2034)